src/status_im/ui/screens/chat/views.cljs

```diff
@@ -651,7 +651,8 @@
                      :show-input?                      show-input?}]
      ;;INPUT COMPONENT
      (when show-input?
-       [components/chat-input-bottom-sheet chat-id])]))
+       [rn/view
+        [components/chat-input-bottom-sheet chat-id] [components/autocomplete-mentions]])]))
```

Comment:
`autocomplete-mentions` should be a part of `chat-input-bottom-sheet` so it should be inside the component

Reply:
yeah i mean it still can be implemented as here, but we need to move `[rn/view` and `[components/autocomplete-mentions]` inside `chat-input-bottom-sheet` so from here it should be as one component, because currently they are related, but `splited` and used  from outside ns